### PR TITLE
Beachball change command - to commit only changefiles 

### PR DESCRIPTION
**Pull request by nickymunga**
1. This change bumps up [workspace-tools package](https://github.com/microsoft/workspace-tools) to v0.13.0. The commit utility in [workspace-tools package](https://github.com/microsoft/workspace-tools) now accepts [additional options](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---only) for the commit command. 
2. This change aims at taking advantage of this; & include the change files in the commit function call `commit -m "change files" {package-root}\\change\\beachball-05571989-7c13-4b9e-b9bf-60c0dea76b11.json`. This will ensure only the specified changefiles(s) are committed while other staged files will be ignored.

### CODE REVIEW

File: src/changefile/writeChangeFiles.ts [hunk 1]
```diff
@@ -40,7 +40,9 @@ export function writeChangeFiles(
 
     stage(changeFiles, cwd);
     if (commitChangeFiles) {
-      commit('Change files', cwd);
+      // only commit change files, ignore other staged files/changes
+      const commitOptions = `${changeFiles.join(' ')}`;
```
Comment (kenotron) [inline src/changefile/writeChangeFiles.ts:44]: There are several things wrong with this:

1. The whole point of the `commit(msg, cwd, options)`'s new `options` param is to pass an array to the command. It should be just `const commitOptions = changeFiles;` according to the work done in `workspace-tools`.
2. Windows has an annoying problem of max length for a command: https://devblogs.microsoft.com/oldnewthing/20031210-00/?p=41553 - so it's a "bad idea" to shove a potentially growing list of changefiles through the command line like that.
3. You already staged the files with very specific files above. Essentially you're already doing what you intended to do. So I'm not sure if this change even solves the problems you're trying to solve. Have you tested your change before this PR?

Look at line 41-44:

```js
stage(changeFiles, cwd); // this line already staged files specifically for this commit: git add file1; git add file2; git add file3
if (commitChangeFiles) {
  commit("change files", cwd); // this line just calls git commit
}
```

In reading this your PR & your description and the workspace-tools change, I don't think you are solving what you intended.

**Review by kenotron — CHANGES_REQUESTED** [review]
I am not sure what problem you're running into with this change? It seems like we already a bunch of work to stage the specific change files. Unless you're actually say that you stage other things during the PR, then you should consider doing `commit("change files", cwd, "change/*.json")` so you don't blow up the Windows max length on commands. (this is why `stage()` doesn't do it all in one command)

File: src/changefile/writeChangeFiles.ts (hunk 1 above)
Comment (kenotron) [reply src/changefile/writeChangeFiles.ts:44]: actually I suggest we maybe should do `const commitOptions = ['--only', '${workspaceRoot}/change/*.json'];`

Comment (p01) [reply src/changefile/writeChangeFiles.ts:44]: Good idea about `const commitOptions = ['--only', '${workspaceRoot}/change/*.json'];`

The goal of this PR is to make `yarn change` only commit the `change/*.json` files even if other files are staged.

**Comment by nickymunga** [comment]
@kenotron I made the suggested changes. Please review when you can.

**Comment by p01** [comment]
Using `--only` is indeed cleaner. Thanks for the change @nickymunga  👍

**Review by kenotron — APPROVED** [review]
Ooops wrong option in the radio :)

**Pull request merged**